scripts/linux/createArtifactInfo.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script is used to create artifact info file.
+
+usage()
+{
+    echo "$SCRIPT_NAME [options]"
+    echo ""
+    echo "options"
+    echo " -outputFolder        output folder for artifact info file"
```

Comment:
[nit] linux "long name" arguments typically use two dashes `--` and are `kebab-case`. For example:

```
echo " --output-folder         output folder for artifact info file"
echo " --build-number          build number of artifact"
```